msal/src/test/java/com/microsoft/identity/client/e2e/tests/mocked/SingleAccountAcquireTokenMockTest.java

```diff
@@ -66,7 +66,8 @@ public void onAccountChanged(@Nullable IAccount priorAccount, @Nullable IAccount
 
             @Override
             public void onError(@NonNull MsalException exception) {
-                fail("No current account found.");
+                fail("Interpreting an exception as no accounts found.  Unexpected exception: " + exception.getMessage() +
```

Comment:
could also do `throw new AssertionError(exception)` and then we won't have to log the exception message and stack trace ourselves.